docs/cudf/source/conf.py

```diff
@@ -74,9 +74,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.19'
+version = "0.19"
 # The full version, including alpha/beta/rc tags.
-release = '0.19.0'
+release = "0.19.0"
```

Comment:
During the course of my rebase with origin/branch-0.19 it seems my IDE changed these from single quotes to double quotes. There were conflicts I resolved and that seems to be where it happened

Reply:
Any chance we can just reset this file to the upstream branch?

Reply:
For sure. I'm going to let CI continue first however and make sure the actual problem is fixed before restarting another CI build with that change

Reply:
Ok, I was wrong. It is actually black that is making the alterations to this file on commit. Given that black is making the changes this should be fine.